textproc/libxmlxx5/Portfile

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           meson 1.0
+
+compiler.cxx_standard 2017
+
+name                libxmlxx5
```

Comment:
I would probably get the source from [GitHub](https://github.com/libxmlplusplus/libxmlplusplus), using the github PG which simplifies the Portfile quite a bit

Reply:
Reason I'm not doing this is, ages ago I once updated a gnome package to use github and at that time the maintainer told me they preferred to keep using the gnome repository.

I'd rather keep it as similar as possible to other gnome (and libxml++) ports, in case somebody else wants to maintain it later on.

Reply:
okay, keeping things similar in very related ports is indeed good as well. I'd just prefer the more readable Portfile when using the gjthub PG; this looks all good I think and nobody else commented so I'll merge this.